Compilation error with Ubuntu 16.04

```
make posix
...
/home/julianoes/src/Firmware/src/modules/ekf2/ekf2_main.cpp: In member function ‘void Ekf2::task_main()’:
/home/julianoes/src/Firmware/src/modules/ekf2/ekf2_main.cpp:496:49: error: ‘isnan’ was not declared in this scope
    if (!isnan(optical_flow.pixel_flow_y_integral) && !isnan(optical_flow.pixel_flow_x_integral)) {
                                                 ^
compilation terminated due to -Wfatal-errors.
```

--- a/src/modules/ekf2/ekf2_main.cpp
+++ b/src/modules/ekf2/ekf2_main.cpp
@@ -493,7 +493,8 @@ void Ekf2::task_main()
 			flow.gyrodata(2) = optical_flow.gyro_z_rate_integral;
 			flow.dt = optical_flow.integration_timespan;
 
-			if (!isnan(optical_flow.pixel_flow_y_integral) && !isnan(optical_flow.pixel_flow_x_integral)) {
+			if (PX4_ISFINITE(optical_flow.pixel_flow_y_integral) &&
+			    PX4_ISFINITE(optical_flow.pixel_flow_x_integral)) {
 				_ekf->setOpticalFlowData(optical_flow.timestamp, &flow);
 			}
 		}

